bpf/bpf_lxc.c

```diff
@@ -1359,7 +1365,9 @@ int tail_ipv4_policy(struct __ctx_buff *ctx)
 			  &proxy_port, from_host);
 	if (ret == POLICY_ACT_PROXY_REDIRECT) {
 		ret = ctx_redirect_to_proxy4(ctx, &tuple, proxy_port, from_host);
+#ifdef ENABLE_CUSTOM_CALLS
 		proxy_redirect = true;
+#endif
```

Comment:
These `#ifdef`s are very noisy. I think we can just drop them since the compiler should be able to optimize out `proxy_redirect`.